cartographer/sensor/point_cloud.h

```diff
@@ -27,26 +27,41 @@
 namespace cartographer {
 namespace sensor {
 
+// Stores 3D positions of points.
+// For 2D points, the third entry is 0.f.
+// Used for unwarped point clouds but not for sensor measurements.
 typedef std::vector<Eigen::Vector3f> PointCloud;
 
+// Stores 3D positions of points with their measurement time
+// in the fourth entry.
+// Time should be measured in seconds, increasing and relative to the most
```

Comment:
I do not understand this comment. The most current, i.e. closest to now measurement is the last which does not have dt = 0'. How about:

Time is in seconds, increasing and relative to the moment 'points[0]' was acquired.